docs/guides/communication.md

```diff
@@ -229,7 +230,7 @@ const pingPongMachine = Machine({
         src: (context, event) => (callback, onReceive) => {
           // Whenever parent sends 'PING',
           // send parent 'PONG' event
-          onReceive((e) => {
+          onReceive(e => {
```

Comment:
seems like maybe you have not reinstalled the project and Prettier v1 has been used to format this file instead of currently used v2